content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -2,226 +2,183 @@
-receive the products.
+To create and manage vendor bills on receipts using the *ordered quantities* bill control policy,
+first go to :menuselection:`Purchase --> Configuration --> Settings`, scroll down to the
+:guilabel:`Invoicing` section, and select :guilabel:`Ordered quantities`. Then, :guilabel:`Save`
+changes.
 
-.. image:: manage/manage05.png
-  :align: center
+Next, go to the :guilabel:`Purchase` app and click :guilabel:`Create` to create a new Request for
+Quotation (RFQ). Add a :guilabel:`Vendor` to the :abbr:`RFQ (Request for Quotation)`, and add
+products to the :guilabel:`Product` lines by clicking :guilabel:`Add a line`. Then,
+:guilabel:`Confirm Order`. Once the :abbr:`RFQ (Request for Quotation)` is confirmed, click
+:guilabel:`Create Bill`.
 
-Purchasing service products does not trigger a delivery order.
-
-Managing Vendor Bills
----------------------
-
-When you receive a vendor bill for a previous purchase, be sure to
-record it in the Purchases application under the **Control Menu**. You
-need to create a new vendor bill even if you already registered a
-purchase order.
-
-.. image:: manage/manage06.png
-  :align: center
-
-The first thing you will need to do upon creating a vendor bill is to
-select the appropriate vendor, since doing so will also pull up any associated
-accounting or pricelist information. From there, you can choose to
-specify one or multiple Purchase Orders to populate the Vendor Bill
-with. When you select a Purchase Order from the list, Odoo will pull any
-uninvoiced products associated to that Purchase Order and automatically
-populate that information below. If you are having a hard time finding
-the appropriate vendor bill, you may search through the list by
-inputting the vendor reference number or your internal purchase order
-number.
-
-.. image:: manage/manage07.png
-  :align: center
-
-While the invoice is in a draft state, you can make any modifications you
-need (i.e. remove or add product lines, modify quantities, and change
-prices).
+On the :guilabel:`Draft Bill`, add a :guilabel:`Bill Date`, and add additional products to the
+:guilabel:`Product` lines by clicking :guilabel:`Add a line` (if needed). Next, :guilabel:`Confirm`
+the :guilabel:`Draft Bill`.
 
 .. note::
-   Your vendor may send you several bills for the same Purchase Order if:
+   Since the bill control policy is set to *ordered quantities*, the draft bill can be confirmed as
+   soon as it is created, before any products have been received.
 
-   #. Your vendor is in back-order and is sending you invoices as they ship the products.
-   #. Your vendor is sending you a partial bill or asking for a deposit.
+On the new :guilabel:`Vendor Bill`, click :guilabel:`Add Credit Note` or :guilabel:`Add Debit Note`
+to add credit or debit notes to the bill; add a :guilabel:`Bill Reference` number; then click
+:menuselection:`Register Payment --> Create Payment` to complete the :guilabel:`Vendor Bill`.
```

Comment:
Good idea to add in more detail and clarification here. I provided a more in-depth rundown of pretty much every available option and field here, so that later in the doc when they pop up again, readers will be familiar.